pkg/skaffold/initializer/build/init.go

```diff
@@ -74,10 +78,19 @@ func (d *defaultBuildInitializer) PrintAnalysis(out io.Writer) error {
 	return printAnalysis(out, d.enableNewFormat, d.skipBuild, d.artifactInfos, d.builders, d.unresolvedImages)
 }
 
-func (d *defaultBuildInitializer) GenerateManifests() (map[GeneratedArtifactInfo][]byte, error) {
+func (d *defaultBuildInitializer) GenerateManifests(out io.Writer, force bool) (map[GeneratedArtifactInfo][]byte, error) {
 	generatedManifests := map[GeneratedArtifactInfo][]byte{}
 	for _, info := range d.generatedArtifactInfos {
-		manifest, manifestInfo, err := generator.Generate(info.ImageName)
+		port := 8080
```

Comment:
do we assume port 8080 is always the default to portforward?

Reply:
yeah, after talking with nick about it we thought this would be okay. Open to any suggestions though! I'm not sure if there's a better way to go about this

Reply:
Aah. got it. Can you remind us what is the manifest generated?
If in the generated manifest, we do create a container port config, this should be ok to do.

Reply:
is it same as in the  test case?
```
kind: Service
metadata:
  name: image1
  labels:
    app: image1
spec:
  ports:
  - port: 8080
    protocol: TCP
  clusterIP: None
  selector:
    app: image1
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: image1
  labels:
    app: image1
spec:
  replicas: 1
  selector:
    matchLabels:
      app: image1
  template:
    metadata:
      labels:
        app: image1
    spec:
      containers:
      - name: image1
        image: image1

```

Reply:
mhm! it's the same